project/Dependencies.scala

```diff
@@ -67,6 +70,10 @@ object Dependencies {
     val refinedCore = "eu.timepit" %% "refined"      % V.refined
     val refinedCats = "eu.timepit" %% "refined-cats" % V.refined
 
+    // webapp
+    // TODO: What's the import needed for %%% to work?
```

Comment:
@davesmith00000 any ideas about this?

Reply:
Is there a clue here perhaps?
https://github.com/japgolly/scalajs-react/blob/2b7f52fb7762a20a5b3ceaabe6e9b6ccf7003e1d/project/Dependencies.scala

Reply:
That seems to use a plugin I don't have: `JSDependenciesPlugin`. I tried the other imports without success.

Reply:
I'm not sure that's it. I'll go back to asking around.

Reply:
Yes, the good people of Scala.js discord concur:
```
armanbilge — Today at 5:13 PM
they also need to be used inside a setting
https://github.com/http4s/http4s/blob/d00a1dcb8d864ec43857ae9491707ce3fbe5f15f/project/Http4sPlugin.scala#L225
see also platform deps import at top of file
```

Reply:
Thank you! Now for some reason `webapp/fastOptJS` stopped working, getting link issues 😮‍💨. I guess it's time to rest and figure it out tomorrow...